app/mailers/notification_mailer.rb

```diff
@@ -12,10 +12,10 @@ def build_message
     @url = work_url(@work_activity.work)
 
     # Troubleshooting https://github.com/pulibrary/pdc_describe/issues/1783
-    if @url.include?("/describe/describe/")
-      Rails.logger.error("URL #{@url} included /describe/describe/ and was fixed. See https://github.com/pulibrary/pdc_describe/issues/1783")
-      @url = @url.gsub("/describe/describe/", "/describe/")
-    end
+    # if @url.include?("/describe/describe/")
```

Comment:
The context for this needed to be clarified for this, as https://github.com/pulibrary/pdc_describe/issues/1783 was not clear in indicating that the issue was only closed given that this *must* remain in the code base until a more optimal solution is identified.